app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  def update_counter
+    user.update(posts_counter: user.posts_counter + 1)
+  end
+
+  def recent_comments()
+    comments.last(3)
+  end
```

Comment:
- [ ] In the project requirements, you are required to return the 5 most recent comments for a given post.

Reply:
Thanks for letting me know @mke2111! Changed ✔